[Bug]: Error: Cannot find module './src/host'

### Version

v11.0.0-rc.0

### Steps to reproduce

1. Clone this repo: github.com:thymikee/jest-preset-angular.git
2. Go to the examples/example-app-v13 directory
```
cd examples/example-app-v13
```
3. edit package.json file and change jest-preset-angular version from `^10.1.0` to `11.0.0-rc.0`
4. Install dependencies
```
yarn install
```
5. run test
```
yarn test
```

### Expected behavior

Tests run

### Actual behavior

Error is thrown

```
ngcc-jest-processor: running ngcc
Error: Cannot find module './src/host'                                                                                  
Require stack: 
- /Users/<user>/Sites/jest-preset-angular/examples/example-app-v13/node_modules/jest-preset-angular/build/ngtsc/reflection/index.js
- /Users/<user>/Sites/jest-preset-angular/examples/example-app-v13/node_modules/jest-preset-angular/build/transformers/downlevel-ctor.js
- /Users/<user>/Sites/jest-preset-angular/examples/example-app-v13/node_modules/jest-preset-angular/build/compiler/ng-jest-compiler.js
- /Users/<user>/Sites/jest-preset-angular/examples/example-app-v13/node_modules/jest-preset-angular/build/ng-jest-transformer.js
- /Users/<user>/Sites/jest-preset-angular/examples/example-app-v13/node_modules/jest-preset-angular/build/index.js      
- /Users/<user>/Sites/jest-preset-angular/examples/example-app-v13/node_modules/jest-util/build/requireOrImportModule.js
- /Users/<user>/Sites/jest-preset-angular/examples/example-app-v13/node_modules/jest-util/build/index.js                
- /Users/<user>/Sites/jest-preset-angular/examples/example-app-v13/node_modules/jest-config/build/getCacheDirectory.js  
- /Users/<user>/Sites/jest-preset-angular/examples/example-app-v13/node_modules/jest-config/build/Defaults.js           
- /Users/<user>/Sites/jest-preset-angular/examples/example-app-v13/node_modules/jest-config/build/normalize.js          
- /Users/<user>/Sites/jest-preset-angular/examples/example-app-v13/node_modules/jest-config/build/index.js              
- /Users/<user>/Sites/jest-preset-angular/examples/example-app-v13/node_modules/jest-cli/build/init/index.js            
- /Users/<user>/Sites/jest-preset-angular/examples/example-app-v13/node_modules/jest-cli/build/cli/index.js             
- /Users/<user>/Sites/jest-preset-angular/examples/example-app-v13/node_modules/jest-cli/bin/jest.js                    
- /Users/<user>/Sites/jest-preset-angular/examples/example-app-v13/node_modules/jest/bin/jest.js                        
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:889:15)                                         
    at Function.Module._load (internal/modules/cjs/loader.js:745:27)                                                    
    at Module.require (internal/modules/cjs/loader.js:961:19)                                                           
    at require (internal/modules/cjs/helpers.js:92:18)                                                                  
    at Object.<anonymous> (/Users/<user>/Sites/jest-preset-angular/examples/example-app-v13/node_modules/jest-preset-angular/build/ngtsc/reflection/index.js:5:27)
    at Module._compile (internal/modules/cjs/loader.js:1072:14)                                                         
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1101:10)                                           
    at Module.load (internal/modules/cjs/loader.js:937:32)                                                              
    at Function.Module._load (internal/modules/cjs/loader.js:778:12)                                                    
    at Module.require (internal/modules/cjs/loader.js:961:19)  
```

### Additional context

_No response_

### Environment

```shell
npx : 1 installé(s) en 2.093s

  System:
    OS: macOS 11.6
    CPU: (12) x64 Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz
  Binaries:
    Node: 14.17.6 - ~/.nvm/versions/node/v14.17.6/bin/node
    Yarn: 1.22.10 - /usr/local/bin/yarn
    npm: 6.14.15 - ~/.nvm/versions/node/v14.17.6/bin/npm
  npmPackages:
    jest: ^27.3.1 => 27.3.1
```

--- a/.npmignore
+++ b/.npmignore
@@ -4,6 +4,7 @@ examples
 
 # sources are inlined
 src
+!build/ngtsc/reflection/src
 
 # Development scripts
 scripts

--- a/scripts/test-examples.js
+++ b/scripts/test-examples.js
@@ -6,8 +6,6 @@ const execa = require('execa');
 const logger = require('./logger');
 const { exampleAppsToRun, rootDir } = require('./paths');
 
-const jestArgs = process.argv.slice(3);
-
 const executeTest = (projectPath) => {
   // we change current directory
   process.chdir(projectPath);
@@ -38,14 +36,8 @@ const executeTest = (projectPath) => {
   });
 
   // then we can run the tests
-  const cmdLine = ['yarn', 'test'];
-  const cmdESMLine = ['yarn', 'test-esm'];
-  if (jestArgs.length) {
-    cmdLine.push('--');
-    cmdESMLine.push('--');
-    cmdLine.push(...jestArgs);
-    cmdESMLine.push(...jestArgs);
-  }
+  const cmdLine = ['yarn', 'test', '--no-cache'];
+  const cmdESMLine = ['yarn', 'test-esm', '--no-cache'];
 
   logger.log('starting the CommonJS tests using:', ...cmdLine);
   logger.log();

